lib/ssl/test/ssl_cert_SUITE.erl

```diff
@@ -98,8 +98,10 @@
          unsupported_sign_algo_cert_client_auth/1,
          longer_chain/0,
          longer_chain/1,
-         duplicate_chain/0,
-         duplicate_chain/1,
+         cross_singned_chain/0,
+         cross_singned_chain/1,
```

Comment:
type-o, should be cross_signed in lines above?

Reply:
Fixed the typo.